buildenv/jenkins/docker-slaves/ppc64le/centos7/Dockerfile

```diff
@@ -106,16 +104,14 @@ RUN wget -O - http://cpanmin.us | perl - --self-upgrade \
   && cpanm Text::CSV \
   && cpanm JSON
 
-# Setup links to GCC-4.8.5
-RUN rm /usr/bin/c++ /usr/bin/cc \
-  && ln -s g++ /usr/bin/c++ \
-  && ln -s gcc /usr/bin/cc
-
 # Install GCC-7.3.1
 RUN cd /usr/local \
   && wget -O gcc-7.tar.xz "https://ci.adoptopenjdk.net/userContent/gcc/gcc730+ccache.ppc64le.tar.xz" \
   && tar -xJf gcc-7.tar.xz --strip-components=1 \
-  && rm -rf gcc-7.tar.xz
+  && rm -rf gcc-7.tar.xz \
+  && ln -s /usr/local/bin/gcc-7.3 /usr/bin/cc \
+  && ln -s /usr/local/bin/gcc-7.3 /usr/bin/gcc \
+  && ln -s /usr/local/bin/g++-7.3 /usr/bin/g++ 
```

Comment:
Is it necessary to symlink default gcc? I.e. does something need a default gcc?

Reply:
I need to a link to cc to install git, but I don't need the rest of the symlinks

Reply:
you could just use a `cc=/usr/local/bin/gcc-7.3` before the git compilation, but I guess this works too